src/Features/Core/Portable/CodeRefactorings/UseNamedArguments/AbstractUseNamedArgumentsCodeRefactoringProvider.cs

```diff
@@ -195,8 +196,8 @@ protected abstract class Analyzer<TBaseArgumentSyntax, TArgumentSyntax, TArgumen
 
         private class MyCodeAction : CodeAction.DocumentChangeAction
         {
-            public MyCodeAction(string title, Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(title, createChangedDocument)
+            public MyCodeAction(string argumentName, Func<CancellationToken, Task<Document>> createChangedDocument)
+                : base(string.Format(FeaturesResources.Add_argument_name_0, argumentName), createChangedDocument)
```

Comment:
nit: we don't generally do this.  We usually just pass the title along and do the stirng.Format from the caller.